CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 All notable changes to this project will be documented in this file using the standards as defined at [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0).
 
+### Version 3.1.0 *(2022-09-08)* 
```

Comment:
I think the release is set for the 15, is it not?

Reply:
Oh well, I was mostly indicating when it was being updated, not the release date. But it makes sense for it to be the release date. 👍🏻

Reply:
I will keep that in mind for the future.